src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+
+	private final int[] NUMBERS = new int[12];
+
+	private int total = -1;
+
+
+	public void countIn(int in) {
+		if (!isFull())
+			NUMBERS[++total] = in;
+	}
+
+
+	public boolean callCheck() {
+		return total == -1;
+	}
+		
+	public boolean isFull() {
+		return total == 11;
+	}
+
+	protected int peekaboo() {
+		if (callCheck())
+			return -1;
+		return NUMBERS[total];
+	}
+
+	public int countOut() {
+		if (callCheck())
+			return -1;
```

Comment:
A crucial part of lab. is missing. See 1.4 and "Temat: stałe i annotacje". Collaborators work has been ignored? Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?